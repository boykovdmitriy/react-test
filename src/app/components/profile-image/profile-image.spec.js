import React from 'react';
import {ProfileImage} from './index'

describe('profile image component', () => {
    it('render profile-image component', () => {
        const imgUrl = 'http://s3-eu-west-1.amazonaws.com/staging-salony/image/profiles/files/000/000/212/thumb/6.jpg?1448538461';
        const wrapper = shallow(
            <ProfileImage thumb={imgUrl}/>
        );
        expect(wrapper).toMatchSnapshot();
    });
    it('render profile-image without required props', () => {
        expect(() => shallow(<ProfileImage/>)).toThrow();
    });
});