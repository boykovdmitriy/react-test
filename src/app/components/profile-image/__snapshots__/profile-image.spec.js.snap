// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`profile image component render profile-image component 1`] = `
<img
  className="image_preview"
  src="http://s3-eu-west-1.amazonaws.com/staging-salony/image/profiles/files/000/000/212/thumb/6.jpg?1448538461"
/>
`;
